src/components/TextArea/TextArea.visual.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import { VisualContainerElement } from '../../../test/VisualContainerElement';
+import { TextArea, TextAreaProps, TextAreaTheme } from './';
+
+class TextAreaVisual extends React.Component<TextAreaProps> {
+  static defaultProps: TextAreaProps = {
+    placeholder: 'Placeholder',
+    onChange: () => {},
+    value:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+  };
+
+  render() {
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
```

Comment:
Again, we can remove all the `mobile` references, if we don't have a design for mobile